README.md

```diff
@@ -1,2 +1,41 @@
 # ai_on_edge_with_fiware
 This is a project to implement an AI on Edge application with FIWARE. 
+
+## Environment Setup
+### Architecture
+![Architecture](/docs/architecture.png)
```

Comment:
The weather station is missing in the architecture =>  it should be included. (this is the overall architecture)